processing file asmFiles/branch.asm (1 of 17)
processing file asmFiles/count_days.asm (2 of 17)
processing file asmFiles/fib.asm (3 of 17)
processing file asmFiles/halt.asm (4 of 17)
processing file asmFiles/jump.asm (5 of 17)
processing file asmFiles/mergesort.asm (6 of 17)
--- memsim.hex	2022-02-20 16:12:56.923362117 -0500
+++ memcpu.hex	2022-02-20 16:12:56.826358463 -0500
@@ -80,60 +80,60 @@
 :04004F0010E00001BC
 :040050000800004A5A
 :0400510003E00008C0
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
+:0400C0000000000339
+:0400C1000000000734
+:0400C2000000000733
+:0400C3000000000732
+:0400C4000000000830
+:0400C500000000082F
+:0400C600000000082E
+:0400C700000000092C
+:0400C8000000000A2A
+:0400C9000000000F24
+:0400CA000000001022
+:0400CB000000001120
+:0400CC000000001719
+:0400CD000000001A15
+:0400CE00000000200E
+:0400CF00000000200D
+:0400D0000000002309
+:0400D1000000002407
+:0400D2000000002604
+:0400D3000000002603
+:0400D4000000002800
+:0400D5000000002FF8
+:0400D6000000005ACC
+:0400D7000000005ACB
+:0400D8000000005ACA
+:0400D9000000005AC9
+:0400DA000000005AC8
+:0400DB000000005AC7
+:0400DC000000005AC6
+:0400DD000000005AC5
 :0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400DF000000005AC3
+:0400E0000000005AC2
+:0400E10000000030EB
+:0400E20000000030EA
+:0400E30000000030E9
+:0400E40000000030E8
+:0400E50000000030E7
+:0400E60000000030E6
+:0400E70000000030E5
+:0400E80000000030E4
+:0400E90000000030E3
+:0400EA0000000030E2
+:0400EB0000000030E1
+:0400EC0000000030E0
+:0400ED0000000030DF
+:0400EE0000000030DE
+:0400EF0000000030DD
+:0400F00000000030DC
+:0400F10000000030DB
+:0400F20000000030DA
+:0400F30000000030D9
+:0400F40000000030D8
+:0400F50000000030D7
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -145,69 +145,37 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401400000000002B9
+:04014100000000308A
+:040142000000003089
+:040143000000003088
+:040144000000003087
+:040145000000003086
+:040146000000003085
+:040147000000003084
+:040148000000003083
+:040149000000003082
+:04014A000000003081
+:04014B000000003080
+:04014C00000000307F
+:04014D00000000307E
+:04014E00000000307D
+:04014F00000000307C
+:04015000000000307B
+:04015100000000307A
+:040152000000003079
+:040153000000003078
+:040154000000003077
+:040155000000003076
+:040156000000003273
+:040157000000004163
+:040158000000004261
+:04015900000000435F
+:04015A00000000445D
+:04015B000000005947
+:04015C000000005B44
+:04015D000000005C42
+:04015E000000005C41
+:04015F000000006438
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (7 of 17)
processing file asmFiles/mult_procedure.asm (8 of 17)
processing file asmFiles/search.asm (9 of 17)
processing file asmFiles/test.branchjump1.asm (10 of 17)
processing file asmFiles/test.control_hazard.asm (11 of 17)
processing file asmFiles/test.data_hazard.asm (12 of 17)
processing file asmFiles/test.ldst1.asm (13 of 17)
processing file asmFiles/test.ldst2.asm (14 of 17)
processing file asmFiles/test.loadstore.asm (15 of 17)
processing file asmFiles/test.rtype.asm (16 of 17)
processing file asmFiles/test.structure_hazard.asm (17 of 17)
